site/docs/plugins/autoquote.md

```diff
@@ -0,0 +1,58 @@
+# Always replying to messages
```

Comment:
Forcing users to reply (`autoquote`)

Reply:
It's the bot that it forced to reply, though. The next sent messages are going to quote the message that triggered the update

Reply:
brb. will look into it tomorrow.

Reply:
Always Replying to Messages

Reply:
```suggestion
# Always Replying to Messages
```